restapi/stories.go

```diff
@@ -190,6 +189,10 @@ func (api api) CreateStoriesFormData(w http.ResponseWriter, r *http.Request) ren
 	if err := api.database.Create(&story).Error; err != nil {
 		return rest.ErrInternal(api.logger, err)
 	}
+
+	// We set the story to be private when it is initially uploaded
+	story.IsPublic = false
```

Comment:
Dont think we would need to this if we set the default value as false in the gorm model (check my other comment)